source/scriptHandler.py

```diff
@@ -35,7 +36,7 @@ def _makeKbEmulateScript(scriptName):
 		# __name__ must be str; i.e. can't be unicode.
 		scriptName = scriptName.encode("mbcs")
 	func.__name__ = "script_%s" % scriptName
-	func.__doc__ = _("Emulates pressing %s on the system keyboard") % keyName
+	func.__doc__ = _("Emulates pressing %s on the system keyboard") % keyLabels.getKeyCombinationLabel(keyName)

```

Comment:
You can instead use `emuGesture.displayName` here. This does more or less the same thing, but it is the same code used to retrieve display names for normal keyboard keys in input help, and since we have the gesture, we may as well use it. There's actually a slight difference in the output too because it accounts for the set ordering that NVDA uses for modifier keys. You could of course do this in your `keylabels` function, but it's not really important.

You still need your `keyLabels` function for the Gestures dialog stuff because we don't build the actual gesture in that case, so we can't use the `displayname` property.